### DISCUSSION TITLE
Ingame Leader Board System

### DISCUSSION THREAD

**Discussion by Z1kou** [sunflower-land/sunflower-land#1210] [Ideas]
*We at Pixelized Crib are exploring new features to add to the game, and today I am gonna open several discussion's for every single one of them so sit down get a drink and buckle up because i need your spicy opinion's everyone !*

 # ISSUE

With the coming of the ingame trading system we will be lacking ingame tool to better do research about the crops and ressources you would like to trade, having to go outside of your game to access simple data like top crops holder is counter intuitive and could lead to inconvienve to players that could be avoided

# SOLUTION

Adding an ingame leaderboard for crops and ressources so a player can easily have access to who is the best player to trade with / could help spot botters easily ( this is also fun for data enthusiast like me )

What are your thoughts Farmers 👨‍🌾 ?

**Comment by Aminbtdds** [comment]
Having ingame leaderboard is great, but i guess it only shows possibility of finding desired resource on leaderboard farms.
Hope at some point we'll have live listings on ingame market.

**Comment by JustJayHSP** [comment]
As much as I'd want that..That  would keep everyone else from being known if there's already a number one Lumber Seller? People will just keep going there due to popularity and how big the resources he can provide.

If there's no farmer name but we can see the top selling crops/materials that would be much better.

Once you click those crops you get a list of Farmers who sell those. Idk if that would work or just easily complicate the coding itself

**Comment by TheDcol** [comment]
Introduces a challenge for hoarders and people who want to be potato king, great idea
